### ISSUE TITLE
iOS: Breaks accessibility

### ISSUE THREAD

**Issue by jgillick** [react-native-menu/menu#659]
I'm having accessibility issues with this component. When using the Accessibility Inspector, it does not read the content of `MenuView` and only reads that there is a button there. In my case, it should say that there is the text "Hello" and an indicator that a long-press action is enabled.

<img width="660" alt="Screenshot 2024-01-03 at 1 45 57 PM" src="https://github.com/react-native-menu/menu/assets/35894/4ce89a45-acf9-44fa-9e8d-6d0b71cec10d">

When using the voice over navigation buttons it skips over the text.

You can see in the node hierarchy (as voice over sees it), the nodes end at the MenuView:

<img width="358" alt="Screenshot 2024-01-03 at 1 46 42 PM" src="https://github.com/react-native-menu/menu/assets/35894/60cc630b-2e53-46da-b2b9-7f79ff409aed">

This is the JSX that I've implemented:

```jsx
<MenuView
  onPressAction={onLongPress}
  actions={longPressActions}
  shouldOpenOnLongPress={true}
  style={baseStyles.container}
>
  <View style={containerStyles} removeClippedSubviews={false} pointerEvents="auto">
    <Text style={baseStyles.text}>{message.content}</Text>
  </View>
</MenuView>
);
```

**Comment by trcoffman** [comment]
Yeah I'm also having this issue.
